components/Ripple/README.md

```diff
@@ -117,11 +139,11 @@ MDCRippleTouchController *rippleTouchController = [[MDCRippleTouchController all
 ```
 <!--</div>-->
 
-
 The `MDCRippleTouchControllerDelegate` gives you some control over aspects of the
-ripple/touch relationship, such as to temporarily disable ripple
-touches, the following code uses the delegate's
-`rippleTouchController:shouldProcessRippleTouchesAtTouchLocation:` method:
+ripple/touch relationship and its placement in the view hierarchy.
+In the below example we are using the delegate to declare we want to only process ripple 
+touches if the touch is in a certain location, and also insert the Ripple view at the very bottom of
+the parent view's view hierarchy:
```

Comment:
Consider explaining why we need to insert the ripple view at the bottom of the parent view's hierarchy here. Good opportunity to educate.